src/app/page.tsx

```diff
@@ -106,6 +106,7 @@ export default function Home() {
         <table>
           <tbody>
             <tr>
+              <th>Jobs</th>
```

Comment:
What if we change this `Jobs` heading? Since this app is not referring jobs and only it is showing companies that offered job offer to abroad employees, maybe we can remove this heading or rename it somethings more relevant. What do you think?

Reply:
Indeed, I agree with you, I think it's sort of redundant (not relevant).
So, I will send another tiny PR.